components/apimgt/org.wso2.carbon.apimgt.core/src/main/java/org/wso2/carbon/apimgt/core/impl/APIPublisherImpl.java

```diff
@@ -2052,6 +2054,46 @@ public WSDLArchiveInfo extractAndValidateWSDLArchive(InputStream inputStream)
         return subscriberList;
     }
 
```

Comment:
Overally we need to add some debug logs to troubleshoot issues. Please add debug logs in appropriate methids